projects/knora-ui/src/lib/viewer/values/date-value/date-input/date-input.component.html

```diff
@@ -0,0 +1,28 @@
+<div [formGroup]="form" class="date-input-container">
+  <mat-form-field class="start child-input-component">
+    <span class="calendar">{{startDateControl.value?.calendarName}}</span>
+    <kui-jdn-datepicker [activeCalendar]="startDateControl.value?.calendarName">
+      <input matInput [matDatepicker]="picker1" [formControlName]="'dateStart'" (dateChange)="_handleInput()" [errorStateMatcher]="matcher" readonly>
+      <mat-error *ngIf="startDateControl.hasError('required')">
+        Start date is <strong>required</strong>
+      </mat-error>
+      <mat-error *ngIf="startDateControl.hasError('sameCalendarRequired')">
+        In a period, start and end date must have the same <strong>calendar</strong>
```

Comment:
change wording from "must have" to "must use" for a clearer error message

Reply:
done in https://github.com/dasch-swiss/knora-ui-ng-lib/pull/35/commits/ec7687d5b89e9937d83375f48610c4cd933e43c8